### PR TITLE
PCSX2-WX: Display the video mode on titlebar

### PR DESCRIPTION
**Pull request by ssakash**
**Summary of changes**:

* Display current video mode at the title bar. (Useful for debugging purposes and for user feedback at cases when video mode is being changed)

### CODE REVIEW

File: pcsx2/gui/AppConfig.h [hunk 1]
```diff
@@ -20,6 +20,8 @@
 #include "CDVD/CDVDaccess.h"
 #include <memory>
 
+const wxString DefaultTitleTemplate = L"Slot: ${slot} | Speed: ${speed} (${vfps}) | ${videomode} | Limiter: ${limiter} | ${gsdx} | ${omodei} | ${cpuusage}";
```
Comment (turtleli) [inline pcsx2/gui/AppConfig.h]: Is this variable necessary? It's only used once.

Note: by default const has internal linkage, so every file that directly or indirectly includes AppConfig.h will have its own copy of the variable. The compiler might be able to optimise it away in a release build, but that probably won't happen in a debug build.

Comment (ssakash) [reply pcsx2/gui/AppConfig.h]: Yeah, I'll be removing it. I initially had it for a different intention but not needed any more.

**Comment by FlatOutPS2** [comment]
Good addition, but the location on the titlebar between Speed and Limiter is not as good.

**Comment by karasuhebi** [comment]
Why is this on by default? I understand it can probably be useful for debugging purposes (so it's nice to have the option to turn it on if needed) but I don't think users care all that much when PCSX2 is on NTSC or PAL. And if they know enough to care about that, they can certainly figure out how to turn the option on.

**Pull request merged**